numpy/core/_add_newdocs.py

```diff
@@ -5327,7 +5327,8 @@
     `__array_interface__` attribute.
 
     Warning: This attribute exists specifically for `__array_interface__`,
-    and is not a datatype description compatible with `np.dtype`.
+    and passing it directly to `np.dtype` will not accurately reconstruct
+    most dtypes (e.g. scalar and subarray dtypes).
```

Comment:
"most dtypes (e.g., scalar and subarray dtypes)" -> "some dtypes (e.g., scalar and subarray dtypes)" with a comma

Reply:
Done